ecl/hql/hqlexpr.cpp

```diff
@@ -8203,7 +8345,19 @@ IHqlExpression *CHqlRemoteScope::lookupSymbol(IIdAtom * searchName, unsigned loo
     {
         StringBuffer msg("Definition for ");
         msg.append(str(searchName)).append(" contains no text");
-        throw createError(ERR_EXPORT_OR_SHARE, msg.str(), filename, 0, 0, 1);
+        if(ctx.queryParseContext().autoIdSuggestions)
```

Comment:
I don't think this is likely to be a useful improvement.  Do you have examples where it is?